src/safecast.yaml

```diff
@@ -0,0 +1,31 @@
+---
+description: >
+  Safecast is a global volunter-centered **Citizen Science** project working to
+  empower people with open data about their environments.
+license: MIT
+name: Safecast
+programmingLanguages:
+  - C++
+  - C
+  - JavaScript
+  - PHP
+  - Go
```

Comment:
There's also quite a bit of [PLpgSQL](https://github.com/Safecast/ingest/search?l=plpgsql) that I'd include